### PR TITLE
Add the Athenaeum links/images

### PR DESCRIPTION
**Pull request by tomudding**
This adds the Athenaeum logo and links to the homepage and education page. This is part of the deal between Athenaeum and GEWIS.

**Homepage:**
GEWIS-web #1018 - Athenaeum Logo Homepage

**Education Page:**
GEWIS-WEB #1018 - Athenaeum Banner Education

### CODE REVIEW

**Comment by tomudding** [comment]
I did not resize the banner because (quoting Wouter) "these changes should be implemented as is".

File: module/Education/view/education/education/index.phtml [hunk 1]
```diff
@@ -114,6 +114,13 @@ $this->headTitle($this->translate('Education'));
                     <div class="panel-heading">
                         <span class="h4"><?= $this->translate('Books') ?></span>
                     </div>
+                    <div class="panel-thumb">
+                        <a class="thumbnail" href="https://mindbus.go2cloud.org/aff_c?offer_id=65&aff_id=639&url=https://www.athenaeum.nl/studieboeken/studieverenigingen/?utm_source=639" style="margin-bottom: 0 !important;">
```
Comment (Koen1999) [inline module/Education/view/education/education/index.phtml:118]: Shouldn't we just make this refer to https://gewis.nl/books. The CBC has already adjusted this link to this one.

Comment (tomudding) [reply module/Education/view/education/education/index.phtml:118]: @Koen1999 I suppose we can do that. I did it this way because this is "as is".

Comment (Koen1999) [reply module/Education/view/education/education/index.phtml:118]: Mmmmhm, also seems reasonable. We'll do the easy way from here

**Pull request merged**